charts/dev/theme-park/values.yaml

```diff
@@ -0,0 +1,36 @@
+image:
+  repository: ghcr.io/k8s-at-home/theme-park
+  pullPolicy: IfNotPresent
+  tag: v1.9.4@sha256:6d576b5aa9c08305026ff76ba9e08e47c40dfce958293f5b3307cae05cff873a
```

Comment:
Also add the image on truecharts/containers (mirror directory)
And later we update the image to point on our repo